### PR TITLE
Security Weapon Overhaul: Ballistic Edition

### PR DESCRIPTION
**Pull request by Steelpoint**
This PR makes multiple changes to the standard and alternative wepaons on offer to security personal, and is based on the large discussion from the thread 'Default Lethal Weapons for Security': https://tgstation13.org/phpBB/viewtopic.php?f=9&t=4472

---

There new weapons are now available to security personal to varying degrees.

Weapon Images
screenshot 2015-09-08 20 00 21

Mark 58 Pistol: New standard issue sidearm issued to all security personal replacing the taser. Holds ten rounds per magazine, default ammo is a non-lethal mag with each bullet dealing 60 stamina damage per round. Lethal ammo is located in the armoury with those bullets dealing 25 damage per bullet.

Mark 58 Officer Pistol: Identical to the standard pistol aside from it having a custom wooden grip, issued to the Warden and Head of Security.

WT-550 Automatic Rifle: A high-alert weapon located in the armoury to be deployed in critical and dangerous situations.  A PDF with a 20 round magazine, with each bullet dealing 25 damage per bullet. Warden is issued one Auto Rifle at round start.

Z8 Marksman Rifle: Powerful semi-auto weapon issued to the Head of Security. Deals 50 damage per round and has five bullets per mag. But suffers from a slow rate of fire and only having one spare magazine on station. (INTEND TO LOWER DAMAGE)

---

Altered Armoury
screenshot 2015-09-08 19 58 30

Main difference is that the bullet proof vests have been moved to the new tac room and have been replaced with three standard Mark 58 Pistols.

---

TO-DO
Add new cargo crates for additional ammo and pistols. (Done)

Balancing!?!

Refinement.

Changelog 

---

Criticism and suggestions are greatly appreciated. This PR is not finished in terms of weapon damage balance but otherwise is fully ready to go.

### CODE REVIEW

**Comment by Steelpoint** [comment]
Link to feedback thread: https://tgstation13.org/phpBB/viewtopic.php?f=10&t=4626

**Comment by Thunder12345** [comment]
+6,887 −62,974
Something isn't right here

**Comment by Steelpoint** [comment]
Its some bug that's going on with map merging as its not working.

**Comment by Thunder12345** [comment]
Ah. Other than that, I very greatly approve. How much pistol ammo is available before you have to get more from somewhere?

**Comment by Steelpoint** [comment]
That's something up for change.

Currently Officers spawn with one spare non-lethal mag.

The armoury is stocked with six lethal and six non-lethal mags.

I will undoublty buff it when I also add the cargo crates in. I may also put some spare mags in the SecVender.

**Comment by Jalleo** [comment]
I personally would rather have the tasers as default sidearm and have the blue alert say that officer can now carry the pistol sidearm.
I dont like how this will make lethal ammo more easily available to standard officers though.

**Comment by Dorsisdwarf** [comment]
I said this on the thread, but why does the HOS now have the most deadly weapon in the game?
800-brute pool of bullet damage (The type people are most vulnerable against) is not to be sneezed at.

Additionally, this PR will have to make eswords and eshields block bullets too, which will result in balance issues.

**Comment by Steelpoint** [comment]
They currently do block bullets, but only at a 50% ratio. I'll look into either buffing it to near 100% or lowering the price of the e-shields to compensate.

Also as noted on the forums the HoS currently has little advantage over a officer barring his slightly better taser (which is just a taser with a laser setting) and that his armour is just as weak as officer armour.

**Comment by ToadStar** [comment]
Why not like, for ammo refills, add in 16-20 clips for stun rounds into the sec-tec, and 5-6 rounds of lethals into the sec-tec? That might alleviate the ammo issue a bit, and it'd make vending machine refills actually get used.

**Comment by Steelpoint** [comment]
What I plan to do for ammo is throw more mags into the armoury but also put several boxes full of dozens of bullets, meaning you have to eventually manually put the bullets into your spent mags, this also encourages officers to not throw away their mags when they are spent.

**Comment by ToadStar** [comment]
I still think having some reloads in either the locker or vending machine might help during the rounds where the warden/HoS are being dumb and won't let you take ammo from the armory.

**Comment by Jalleo** [comment]
Oh god that idea is so awful it will encourage sec to just switch to energy weapons later in the round.

**Comment by Steelpoint** [comment]
I was thinking of putting the ammo boxes in the common sec area for officers to freely access.

**Comment by ToadStar** [comment]
That's much better, some spare empty mags knocking around might not be a bad thing either.

**Comment by MrPerson** [comment]
>What I plan to do for ammo is throw a few more mags into the armoury but also put several boxes full of dozens of bullets, meaning you have to eventually manually put the bullets into your spent mags, this also encourages officers to not throw away their mags when they are spent.

This is, quite literally, the dumbest idea I've heard all year.

**Comment by ToadStar** [comment]
If it's like the current ammo boxes, it's as simple as knocking the ammo box against the magazine and it'll reload. Making these boxes come in boxes/dufflebags filled with them and having like, 40 bullets to a box, along with a dozen or so empty mags would probably do really well.

Of course, adding a lot more ammo into the armory and some pre-filled mags might not hurt, either.

**Comment by Steelpoint** [comment]
Think of it like this.

Officers spawn with their gun and one spare mag, that's two magazines which equals 20 bullets, that's a potential of ten stuns.

After they've expended their ammo they come back to security and take new bullets out of the general ammo box to refill their now empty mags.

Also its real easy considering, as said above, you just knock the mag on it and it will fill up the gun with the bullets.

**Comment by ToadStar** [comment]
It's more the inverse, since clicking a mag against an ammo box will empty the ammo into it, iirc. Not sure.

**Comment by MrPerson** [comment]
But then why not just, you know, have more mags?

**Comment by Zilenan91** [comment]
You don't always use all the bullets in a mag.

**Comment by Steelpoint** [comment]
Because then Officers can easily carry a lot of mags on them. At least restricting them to two mags initially means its not a case of a officer having a lot of ammo to toss away.

**Comment by ToadStar** [comment]
The sec belt also holds ammo mags, as unknown as that is (surprisingly). Granted, it's not much with five slots, but eh.

**Comment by MrPerson** [comment]
Actually the real question is, why are you giving them ballistic weapons at all? One of the very few themes we have going in the game is that Nanotrasen uses energy weapons and the Syndicate uses ballistic ones. That's not even getting into the fact bullets are fiddly as all shit compared to energy weapons, and we REALLY don't need to have rechargers and bullet reloading when having just one system will do.

**Comment by Dorsisdwarf** [comment]
>not much with five slots
5x10 + 1x10 (in the gun) + 5x10 (Internals box) = 110 shots without having to go hide in the brig and reload

**Comment by Steelpoint** [comment]
Hence why they only have two mags each.

**Comment by xxalpha** [comment]
>giving security bullet guns
>giving hos 80 brute gun

I know you have a boner for security, but this is just meme tier.

**Comment by Steelpoint** [comment]
It can be scaled back if we want, damage numbers are not set in stone so give out suggestions.

**Comment by xxalpha** [comment]
Don't give these to security at all because it will fuck over traitors.

**Comment by Jalleo** [comment]
25 brute

**Comment by Steelpoint** [comment]
How will it fuck over traitors specifically @xxalpha? The new pistol takes two hits to stun someone so at least its not a one hit stun. The Auto Rifle is a obvious gun that takes effort to acquire and has limited ammo and the Z8 is a HoS specific gun of which I'm sure a idiot hos will get it slipped out of his hands more than he uses it well.

Also @Jalleo the pistol and rifle deal 25 dam already, are you saying 25 for the marksman rifle?

**Comment by Jalleo** [comment]
Yes

**Comment by Steelpoint** [comment]
That would kinda make the marksman rifle redundant if it was a worse option than the auto rifle.

Perhaps 50 damage or similar so at least it has a edge.

**Comment by xxalpha** [comment]
>How will it fuck over traitors specifically

Let's see:
 - Can't EMP Security firepower because they are not energy guns
 - Esword won't block the bullets making it useless
 - E I G H T Y B R U T E D A M A G E

**Comment by Jalleo** [comment]
Auto-rifle can have detective tier damage since they would probably be shooting automatically normally.
Remember this would make revs easily kill-able if it was really really strong in damage.

**Comment by Zilenan91** [comment]
Perhaps locking the Auto Rifles under Red Alert?

**Comment by MrPerson** [comment]
Why not just give sec energy weapons instead of disablers or tasers? Come on now. It's easy.

**Comment by GunHog** [comment]
This change is far too disruptive and goes against how Security and Antagonists are balanced.

Security should not have any lethal weapons by default, they should be required to arm up if they confirm a threat that requires it. Furthermore, this severely diminishes the tactic of looting or destroying the armory to prevent an immediate and heavy response to enemy activity.

Security should certainly not have ballistic weaponry so widely supplied and certainly not a default choice, barring the armory. There are antagonist items and mobs balanced around resisting Security's weapons (Juggernaut, Nuke Ops' energy shield, energy swords, etc), which is severely nullified by this change.

Fighting them with EMP based weaponry and abilities is now entirely pointless, as ballistic weapons are not affected by them.

They now even have less reason to depend on Cargo and/or Science for upgraded weaponry or more supplies - they are entirely self sufficient with the exception of ordering more implants from cargo.

This will also encourage even stronger 'validhunting' behavior by Security and greatly reduced effectiveness of human antagonists, two things which have caused public outcry due to them being too afraid and inactive in rounds.

Significant consideration of the balance changes this PR will cause is greatly recommended.

**Comment by Zilenan91** [comment]
Gun Hog they don't have default lethal weapons, they have a pistol that shoots stamina rounds, with only about 6 lethal mags in the armory itself.

**Comment by Jalleo** [comment]
I request a FUCK tag @WJohn

**Comment by Steelpoint** [comment]
I should emphasize that Officers do not start with any lethal ammo, they have to get that from the armoury proper.

Secondly saying that this will make security more deadly or hostile is ignoring that fact that the armoury is currently stocked with laser and energy guns which are just as deadly as the auto rifles. The addition of auto rifles won't change anything in that regard.

Also this change arguably makes security more dependant on cargo due to their ammo having a finite limit instead of being unlimited. 

I would love to know why you think this will encourage stronger validhunting, especially considering that's their job in the first place.

I can see what your saying about antagonist resistances to energy versus ballistic weapons, but this is easily addressed with number changes (also jugs already tank a lot of damage irrespective of it being ballistic or energy).

**Comment by WJohn** [comment]
I am not a fan of this. That and it also makes the warden's area super insanely cramped, and looks really bad with the armory. Actually change the armory up, would you?

Also I thought the entire point of having laser weapons was thematic, they have lasers to avoid doing damage to the station.

**Comment by Kelenius** [comment]
Everybody check out, Steelpoint is buffing security again.

Did he already use "traitors can steal it so it's fair" as an argument already?

**Comment by Steelpoint** [comment]
Kelenius I think this PR is the first honest to god buff to security I've ever made barring the HoS gun (or the armour changes) which honestly was less a buff and more giving him a actual good weapon.

Keep it to singulo please.

**Comment by Jalleo** [comment]
@Steelpoint this actually can be a buff to traitors at the same time due to meta we just need proper balance if that is not done this will break your PR

**Comment by xxalpha** [comment]
Security doesn't need this. Why are you giving them more weapons, weapons that will disrupt balance?

**Comment by Steelpoint** [comment]
Hence why I'm fully open to balance changes, but I'm not making any changes until we've gotten more discussion out of this. Better to do the changes in one swoop than constant changes which disrupt the conversation.

@xxalpha You can say that for any change to the game. Why should they need or get this? 

It won't disrupt balance if its, surprise, balanced correctly!

**Comment by xxalpha** [comment]
>You can say that for any change to the game.

Like you said for cyborg tasers.

Security doesn't need this, at all.

**Comment by Jalleo** [comment]
Can ya come on IRC so we can have a much more effective faster chat @Steelpoint

**Comment by WJohn** [comment]
I don't think you understand how combat even functions in this game if you're seriously planning on adding an 80 brute gun which also does enough stamina damage to instantly knock someone over.

For all intents and purposes, that's a one hit kill, since ALL combat in this game revolves around striking first. All you do is shoot once, melee a couple of times, and you've critted them. Move on to the next or so people.

**Comment by GunHog** [comment]
Specifically, I feel they will take a more lethal approach because it is much easier to pack more dangerous weapons. I cannot help but think them having more weapons like this, and especially powerful HoS weapon will make them more prone to on-the-spot executions. It is purely conjecture, though.

Even so, I cannot stress enough how much balance will be destroyed by this. What are you hopping to achieve, if you are going to buff the antags that previously had a counter to their energy weapons? Also, how are you going to deal with the fact that this makes EMPs ineffective against Security officers?

**Comment by CosmicScientist** [comment]
As an officer, to be fully kitted you need:
Taser
Stun baton
At least one pair of cuffs, cables or zipties
Flash
Pepper spray
Torch

After this you'll also need a pistol and spare magazines, so that's pushing into using the bag and box unless their belt gets upgraded. This isn't counting carrying around a book of space law for fluff, carrying an evidence box just-in-case or extra cuffs because you will need them.

**Comment by Steelpoint** [comment]
@WJohn I did clearly state that I'm nerfing the Z8. I also stated that I'm observing damage numbers, 80 damage was just a placeholder as I've described mutliple times.

@GunHog EMP's still counter most officers equipment barring the gun (also emping tends to stun people from the headset overloading). I'm still going to be examining antag items/armour for balance changes needed but honestly I have rarely seen EMP's used against officer to date.

**Comment by Zilenan91** [comment]
I do agree with WJ that the wardens office is not going to look the best after this. Perhaps a better place for the Auto Rifles would be under shutters or behind reinforced windoors on the right side of the armory room, a tile left of the reinforced table above the air scrubber. There wouldn't be as many auto rifles, but it would look a lot nicer, and any alternatives would be completely redesigning the Armory to compensate for these.

Also @cosmicscientist this PR should remove tasers from round start officer equipment. The only ones left will be the ones in the armory.

**Comment by Steelpoint** [comment]
I'll look into a better redesign of the armoury.

**Comment by Steelpoint** [comment]
Worst case I can remove the pistols and only issue one to the Warden and HoS and slap a few spares in the armoury if the pistols are too effective.

**Comment by Chocobro** [comment]
the first two magazines in the image look like garbage to me i'm not sure what specifically but it's there
otherwise :+1:

**Comment by Jalleo** [comment]
@Steelpoint can ya come on IRC so we can have a faster and more effective conversation since most of us are there.

**Comment by Zilenan91** [comment]
The marksman rifle for the HoS is probably too much imo, it's just too effective vs. certain antags to be good for the game as a whole.

**Comment by Steelpoint** [comment]
I could just make it a semi-auto shotgun, essentially a slightly better riot shotgun.

**Comment by MrPerson** [comment]
The pistols were the ENTIRE FUCKING PURPOSE of the request to begin with! Removing them means you're just changing shit without any real reasoning as to why.

**Comment by Steelpoint** [comment]
No I went into this with the intention of adding the three weapons listed above.

**Comment by Zilenan91** [comment]
I don't think the HoS even needs the buff, it would still be very strong even if you added the shotgun.

Then again, if it didn't come with lethal rounds at round start then it would be fine, but putting it in the armory as a 1 of would be preferrable instead of in the HoS office.

@Mrperson Steelpoint was saying that removing them from officers at round start (aka not spawning with them) would be what he'd do if they were too good.

**Comment by CosmicScientist** [comment]
@Zilenan91 I didn't see that. I don't like that.

Either this means playing as a Warden/Wardenless HoS will mean handing out tasers to everyone for five minutes or having them ill equipped. Actually besides the immediate stun of a taser which will only help against players without e-swords or similar unless I'm mistaken, why does the taser even exist?

60 Stamina versus whatever a disabler shot does means... I guess removing the 1% of times players get away after not dodging one hit because they'll be too slow to even try? The only difference is how to resupply, back to security or a recharge station and I fail to see why a recharge station is better.

**Comment by Steelpoint** [comment]
I've put forward the idea on IRC to give officers a mining voucher that can give them either a pistol or taser.

**Comment by Zilenan91** [comment]
Honestly I think tasers should just be completely phased out and be replaced by the pistols, aside from the ones in the armory of course.

**Comment by AlexanderUlanH** [comment]
The only balance change to security that could be accomplished by pistols but couldn't be accomplished by changing the projectiles available to security's main weapon is dependence on cargo or the warden for ammunition.
In exchange for that, you give up the established flavor of no-collateral-damage NT energy weapons and highly destructive Syndicate projectile weapons, and you give up the established balance of EMP's/eswords/eshields.

Unless cargo/warden dependence is absolutely balance critical, you can get around all of that by just giving security an energy weapon or a taser setting that does what the projectile weapon would have done.

**Comment by Incomptinence** [comment]
80 brute seems pretty superficial. Revolver handles the 2shot crit just fine with 70 brute.
@CosmicScientist Does one dissabler hit cause slowdown? I don't think so?

**Comment by Zilenan91** [comment]
@Incomptinence As far as I know how disablers work is the reason they slow you is because they're lowering your health by using a damage type that is constantly regenerating, so any slowing they have is simply due to your lowered max health.

Also the marksman rifle will be a HUGE problem for antags like wizards. This sound clip http://sam.wileycomputerworks.com/SS13/Wiz.wav will be more accurate than ever, because having 80% of a wizards health (and a good bit of bleeding) gone in a single shot from just one dude will really hurt the mode.

**Comment by Jalleo** [comment]
@Incompitence its being changed and the 80 damage is with 55 stamina damage btw

**Comment by Incomptinence** [comment]
@Zilenan91 

I do suppose more wizards are choosing not to summon guns now they are onto other gimmicks so they basically shouldn't be getting revolver level damage against themselves by default. 

The enormous stamina damage in addition is the real issue but 80 isn't all that amazing on it's own.

**Comment by Steelpoint** [comment]
Look just forget that it has 80 damage, its going to get nerfed.

**Comment by Incomptinence** [comment]
80 damage slow rate of fire is fine really. 

Being basically a taser shot with 4 laser shots brute equivalent mixed into it is very silly.

**Comment by Steelpoint** [comment]
I intend to remove the stam damage. It'll only deal pure damage.

**Comment by Zilenan91** [comment]
The problem with it is that having anything at that high a damage per shot available round start to non-antags is a little ridiculous though, it'll really screw over solo-antags even if its damage is nerfed considerably.

**Comment by MrPerson** [comment]
>No I went into this with the intention of adding the three weapons listed above.

But what purpose do they serve? The pistols are useless if they're armory only, the autorifle is a laser gun, and the HoS weapon could just as easily be a strong laser gun.

Why not just make these energy weapons?

**Comment by Miauw62** [comment]
literally why.

**Comment by Core0verload** [comment]
:-1: Fuck no. I played with a version of it, and it was unbalanced shit. I see it's new stats. It is still unbalanced shit.

**Comment by Niknakflak** [comment]
:-1:

**Comment by Strathcona** [comment]
I think making them energy weapons with identical stats would be a much better change since it wouldn't involve messing with the energy weapon counterplay many antags rely on. Additionally it would fit in with what NT uses currently. I don't see why they need to be ballistic.

**Comment by Steelpoint** [comment]
Making them energy based makes this PR utterly pointless.

**Comment by Steelpoint** [comment]
Minor change, made pistol a wclass=3 item and lowered rifle dam to 50 with no stam damage.

Keep the feedback coming.

**Comment by Supermichael777** [comment]
actually disables with exchangeable externally recharged magazines might be cool.
swap your battery voltage to go from stun to kill.

**Comment by Steelpoint** [comment]
As I said on the rapidly moving IRC I think to propose that this be merged, when finished, for a week and to gather in game feedback and opinion on the changes,

if the change is horid I will revert the change after a week.

**Comment by Niknakflak** [comment]
Or we could just not merge this in the first place

**Comment by Dorsisdwarf** [comment]
I'm with niknak here. 
this PR fucks over antags, balance, the one bit of the lore we don't trample all over every five minutes, and like half the gamemodes,

**Comment by Steelpoint** [comment]
As is being discussed on IRC that's not the case.

**Comment by Miauw62** [comment]
As is being discussed on IRC that's definitively the case.

**Comment by Supermichael777** [comment]
random thoughts:
the design of the pdw is like that of the prototype smg. cross compatible mags wouldn't hurt.
now my murder-squad can unload 200 rounds at a perp like real cops.
simultaneously nerfing instantaneous take-down with longer lasting power.
at 60 a hit they aren't dodging the second shot.
loading the rounds right and this will be the most ridiculous weapon in the game if you can put different kinds of bullets into the same mag.
what is this lo-or people keep taking about. I never  read any lore page that said that.

**Comment by Ergovisavi** [comment]
Not a fan in the slightest. Security already has access to lethal weaponry, all this does it muck up the current "theming" of energy weapons, on top of causing issues with our already poorly thought out armor values.

And in the end, a taser still beats everything. The hell is the point?

:-1:

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -244,6 +244,30 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
...
+					/obj/item/ammo_box/magazine/stun45,
+					/obj/item/ammo_box/magazine/stun45,
+					/obj/item/ammo_box/magazine/stun45,
+					/obj/item/ammo_box/magazine/stun45,
+					/obj/item/ammo_box/magazine/stun45/lethal,
+					/obj/item/ammo_box/magazine/stun45/lethal,
+					/obj/item/ammo_box/magazine/stun45/lethal,
+					/obj/item/ammo_box/magazine/stun45/lethal,
+					/obj/item/ammo_box/magazine/stun45/lethal)
+	cost = 20
```
Comment (Supermichael777) [inline code/datums/supplypacks.dm]: That's a hundred rounds of ammo for 20 points.
Kinda low-balling 1250 damage and 100 stuns.

File: code/datums/supplypacks.dm [hunk 2]
```diff
@@ -328,6 +352,23 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
...
+	containername = "auto rifle crate"
+
+/datum/supply_packs/security/armory/wt550ammo
+	name = "WT-550 Rifle Ammo Crate"
+	contains = list(/obj/item/ammo_box/magazine/wt550m9,
+					/obj/item/ammo_box/magazine/wt550m9,
+					/obj/item/ammo_box/magazine/wt550m9,
+					/obj/item/ammo_box/magazine/wt550m9,
+					/obj/item/ammo_box/magazine/wt550m9)
+	cost = 30
```
Comment (Supermichael777) [inline code/datums/supplypacks.dm]: this translates to 2000 damage of sweet automatic fire.

File: code/modules/projectiles/projectile/bullets.dm [hunk 3]
```diff
@@ -20,10 +23,16 @@
 /obj/item/projectile/bullet/weakbullet3
 	damage = 20
 
+/obj/item/projectile/bullet/weakbullet4
+	damage = 25
```
Comment (Supermichael777) [inline code/modules/projectiles/projectile/bullets.dm]: deals more damage then the prototype smg roundstart

**Comment by Steelpoint** [comment]
I intend, assuming this is not closed, to code a voucher system allowing for Officers to select between a Taser and a Pistol.

Each weapon has their own advantages and disadvantages.

Taser
* One shot stun
* Rechargable, easier to supply in the long run
* Ability to fire through windows with disable ability
* Limited to seven tile range
* Has to recharge at base after five shots
* Blocked by energy swords/shields

Pistol
* Two shot stun
* Magazine fed, allowing you to reload in the field instead of retreating.
* Unlimited range
* Limited in ammo supply in long run, requires maintenance of a supply chain.
* Capable of piercing shields and swords
* Can swap out to lethal ammo.

File: code/datums/supplypacks.dm (hunk 1 above)
Comment (Steelpoint) [inline code/datums/supplypacks.dm]: All those cost values are subject to change, they are not final. What cost do you think is a good cost?

**Comment by MrPerson** [comment]
No projectiles should be infinite range, that just encourages shooting people off screen. Which is bad.

**Comment by Steelpoint** [comment]
That's for another PR then since all projectiles barring the taser have the range limitation.

**Comment by TechnoAlchemisto** [comment]
Stuns from stamina damage are vastly superior and more debilitating then taser stuns. They keep you cripple you afterwards, and they are not affected by adrenals. Stamina damage is kind of an unfinished system with no real counter at this moment besides entirely dodging or blocking(in energy weapons case) the source. 

This is a thematic and balance issue, although it seems like a great idea, and I agree it's pretty cool, it just isn't right for the server at this point. 

If you want to integrate ballistics you need to do a full overhaul encompassing multiple game modes, items, and even the medical system.

**Comment by Stickymayhem** [comment]
Oh my god there's so many things wrong with this that I'm too lazy to write them all up.

For a start the map changes are a big buff. More r-walls means bombs as an entry point for nuke ops and loud traitors are even less effective than they were previously. The missing side window is now gone and the only entry point is a direct line of fire from the door of the equipment room, making juking and harassing security basically impossible now.

The armory will just get raided by miners to kill everything on the asteroid easily, almost every antagonist is horrendously affected by this.

I've never seen a more blatantly terrible buff before. This is totally unrestrained and out of whack with the balance of the game. the values are wrong, the strategic advantages of weapons are wrong (No returning to base because lol ammo clips, stamina damage for long term stuns, shooting people off screen, like five other things).

I mean jesus fuck steelpoint usually you think

**Comment by Stickymayhem** [comment]
I don't see any way to fix this. It's an uninteresting outright buff and, as is surprisingly rarely the case, the entire core concept is horrible for the game.

It's a shame but this whole thing is an awful idea.

**Comment by Supermichael777** [comment]
i think the core concept is solid but its too much all at once. Three new officer guns and no counterplay is bad. Maybe if syndies got access to some cool laser gimmick thing it would be good. like a laser derringer or a standard laser gun.

**Comment by Stickymayhem** [comment]
The core concept is kind of pointless though.

"Let's add ballistics"
"Why?"
"I don't know but we can just fix everything else in the game to make it work."

**Comment by Nanotransistor** [comment]
This is highly unnecessary, security is more than equipped to deal with every single threat currently faced by the station. As pointed out, you would have to buff and change every single antag to match this additional firepower from sec. Think about 10 nuke ops from 5, twice the amount of starting traitors and lings, twice the TC, etc. It completely fucks up the fairly bad balance that we already have.

We should really re add traitor sec, you would see how all these ridiculous buffs to sec suddenly disappear.

**Comment by Anonus** [comment]
Ah yes, random ballistic weapons. Just make them fire ENERGY BULLETS, so they fit in with nanotrasen's energy weapon thing. They can still use ammo, but they can be ENERGY BULLETS instead.

**Comment by Fox-McCloud** [comment]
Interesting idea, in practice, but I think it needs more work for this to be long term viable.

:+1:  On the idea, but needs more work.

File: code/datums/supplypacks.dm (hunk 1 above)
Comment (Supermichael777) [inline code/datums/supplypacks.dm]: i would actualy keep that cxost and cut it down to 2 of each kind of clip

**Comment by optimumtact** [comment]
should I just take the pistols out of this pull and make a seperate one with them?

**Comment by Aloraydrel** [comment]
This kills the balance :^(

**Comment by Stickymayhem** [comment]
no the special snowflake HoS cannon is shit too

the rifles are too easily accessible and would break mining and are still too powerful. It would ruin every side antag

**Comment by Zilenan91** [comment]
I could see a PR with just the pistols working quite well.

**Comment by Xhuis** [comment]
:-1:

I could see this being extremely controversial. The only way we fixed the complaints about stamina security is when we added hybrid tasers instead of just issuing disablers.

**Comment by Incomptinence** [comment]
@Stickymayhem
While the z8m was originally poorly though out. It is now a 50 damage no stam gun more reasonable any armour will slow it down.

Also a head getting a snowflake item? That is the one drat place unique single issue items are meant to go. The other heads are bursting with unique personal gear just look at the cmo.

**Comment by Stickymayhem** [comment]
The HoS has so much extra shit precisely because steelpoint and others has provided it.

Unique Gun, Several unique gear sets, Unique riot shield, Unique mask.

Where did I say I cared about snowflake. I care about balance and I've got a pretty sturdy handle on it being one of those cunts who uses everything possible to kill everyone. Giving the HoS a massive fucking sniper rifle is a shit idea. It kills pretty much anything in two shots, bypasses shields, and can basically down a whole nuke op team from roundstart. It's ridiculous, unnecessary and I'm sorry Steelpoint spent time on it because he normally does good shit.

**Comment by Lumipharon** [comment]
This is a pretty awful idea.

Put the guns into the code by all means. Even make them obtainable via cargo, or armoury or whatever.
Just dont replace default energy weapons with this ballistic shit.

**Comment by SconesC** [comment]
I like the core concept but the playerbase at large is not mature enough to be given this

Medical system is super dull and as a result more gunplay isn't interesting or fun

HoS gun is a one shot kill, laser someone afterwards and they go into crit

This isn't getting merged

@optimumtact The pistols are very much salvageable and when this inevitably gets closed you really should pull them

**Comment by Stickymayhem** [comment]
What happened to my original idea of shotguns in protected cases.

I thought that was an elegant solution to this perceived problem.

**Comment by Zilenan91** [comment]
@Stickymayhem Giving sec access to free revolvers is a horrible idea.

**Comment by Stickymayhem** [comment]
they were spread so there would be one in each security checkpoint, coming with limited (Actually limited) lethal ammunition. Red Alert would lift bulletproof shields and a security ID would open it. Hardly free considering the coordination required by heads.

**Comment by Iamgoofball** [comment]
>11 hours
>100+ posts

Jesus christ you have to have broken one of my records here

**Comment by Fox-McCloud** [comment]
I tend to agree that this may be best handled on small scale...I *REALLY* like the idea of the pistols--more than anything else.

**Comment by Incomptinence** [comment]
@Stickymayhem
With new 50 damage no stam value it probably won't two shot ops they do have armoured rigs.

**Comment by Steelpoint** [comment]
I was thinking of lowering the marksman rifle to dealing 25 damage or so, but deal 20 stamina damage AND have 50 or so AP capability (meaning it'll ignore most bulletproof armour).

Meaning it'll deal guaranteed 25or so damage against even someone wearing bulletproof armour. I would also give it a ten round mag but also keep its firerate at about a 2 to 3 second refire rate.

--------

I'll work on the voucher system and put in a few ammo boxes in the general security area, I should note that many antagonists armour already offer good bullet protection HOWEVER I will look around and buff them to match or be better than current laser defence values.

**Comment by SconesC** [comment]
I don't see the purpose of the marksman rifle. I would rather it be a singular armory item than an HoS item. It really seems to be unnecessary on top of all the other things being added in.

I think people are more concerned about unarmored antagonists.

**Comment by Steelpoint** [comment]
I looked and current Nuke Op (and Abductor) armour already offers 50 bullet protection, in contrast to it offering 30 laser protection.

@SconesC Ausops did suggest adding a somewhat cheap traitor 'armour vest' that offers basic protection and is invisible without shift-clicking to examine.

I'm dropping some minor buffs to some antag armour to match current or exceed laser protection values. The new values are, so far:

Wizard RIG: Buffing Bullet and Laser defence to 40.

Changeling Armour: Buffed both Bullet and Laser to offer 30 defence

**Comment by MrStonedOne** [comment]
[21:27:28] \<Mrs_Sybil\> AUTOMATIC ANNOUNCEMENT : PR test job started. Updating master then merging PR #11662 locally

**Comment by Steelpoint** [comment]
I think I'm going to branch this PR into two seperate ones, one dealing with the Pistols and the other with the AutoRifle/Marksman rifle.

**Comment by Steelpoint** [comment]
Also right now this PR lacks the map changes made, so it only includes the pistols given to officers and the one auto rifle in the Warden's locker/marksman rifle in the HoS's locker. Meaning ammo's going to be more a problem,.

**Comment by MrStonedOne** [comment]
god damn it, 

fine, i'll revert the update for now

**Comment by Steelpoint** [comment]
I'm opening a new PR to deal with the two changes individually, so they can be judged on their own merits instead of getting lumped togther.

**Comment by Steelpoint** [comment]
@MrStonedOne If you want you can test the pistol PR.

File: code/datums/supplypacks.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/datums/supplypacks.dm]: >implying you'll hit all shots

**Pull request closed without merging**